engine/access/rpc/backend/backend.go

```diff
@@ -144,6 +147,10 @@ func New(
 			log:               log,
 			maxHeightRange:    maxHeightRange,
 		},
+		backendExecutionData: backendExecutionData{
```

Comment:
I think the intention was to instantiate this as a completely separate API on a different port. This appears to be integrating it as an additional API on the existing gRPC interface.